prow/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM quay.io/openshifttest/python:3.9
 
 LABEL vendor="Red Hat Inc." maintainer="OCP Perfscale Team"
 
-RUN curl -sSL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.15/openshift-client-linux.tar.gz  | tar -xvzf -  &&\
+RUN curl -sSL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux.tar.gz | tar -xvzf -  &&\
```

Comment:
We had issues with `GLIBC` as such the fix was dropped.
https://github.com/cloud-bulldozer/e2e-benchmarking/pull/713